remill/Arch/X86/Semantics/BINARY.cpp

```diff
@@ -491,7 +491,7 @@ IF_64BIT(MAKE_DIVxax(rdxrax, REG_RAX, REG_RDX, REG_RAX, REG_RDX))
 
 MAKE_IDIVxax(ax, REG_AL, REG_AH, REG_AL, REG_AH)
 MAKE_IDIVxax(dxax, REG_AX, REG_DX, REG_AX, REG_DX)
-MAKE_IDIVxax(edxeax, REG_EAX, REG_EDX, REG_XAX, REG_XDX)
+MAKE_IDIVxax(edxeax, REG_EAX, REG_EDX, REG_EAX, REG_EDX)
```

Comment:
In 64-bit, the dest regs need to be `rax` and `rdx`, hence the use of `REG_XAX` and `REG_XDX`.